core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -757,18 +757,14 @@ object KafkaConfig {
 
   }
 
-  def configNames() = {
-    import scala.collection.JavaConversions._
-    configDef.names().toList.sorted
-  }
+  def configNames() = configDef.names().asScala.toList.sorted
```

Comment:
`configDef.names()` gives a Set, before the change there was an implicit conversion on the result of names() to convert it to a Set, now its just explicit.